lib/pure/parsecfg.nim

```diff
@@ -564,7 +556,7 @@ proc replace(s: string): string =
 proc writeConfig*(dict: Config, stream: Stream) =
   ## Writes the contents of the table to the specified stream.
   ##
-  ## **Note:** Comment statement will be ignored.
+  ## .. note:: Comment statement will be ignored.
```

Comment:
how about a single PR to change all such instances repo-wide? would make it easier to review
(ok for this one though)

Reply:
well, in the future PR.